deploy/07_giveaway/18_deploy_multi_giveaway.ts

```diff
@@ -0,0 +1,20 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+  const {deployer} = await getNamedAccounts();
+
+  await deploy('Multi_Giveaway_1', {
+    contract: 'MultiGiveaway',
+    from: deployer,
+    log: true,
+    skipIfAlreadyDeployed: true,
+    args: [deployer],
+  });
+};
+export default func;
+func.tags = ['Multi_Giveaway_1_deploy'];
```

Comment:
if the deployment of a contract require an extra setup phase, the convention is to have 2 tags for each
deploy script : ["Multi_Giveway_1", "Multi_Giveaway_1_deploy"]
setup script :   ["Multi_Giveway_1", "Multi_Giveaway_1_setup"]

this allow to ensure both are run via "Multi_Giveaway_1"